packages/compass-utils/src/is-action.ts

```diff
@@ -0,0 +1,15 @@
+// Types duplicated from redux.
+interface Action<T = any> {
+  type: T;
+}
+
+interface AnyAction extends Action {
+  [extraProps: string]: any;
+}
+
+export function isAction<A extends AnyAction>(
```

Comment:
or should i rename it to `isReduxAction`?

Reply:
Just my 2 cents, but I think we're going to be fine with sticking to `isAction` unless we plan on introducing yet another state management library with its own `Action` concept 🙂